utils/merge_syft_sbom.py

```diff
@@ -118,6 +118,27 @@ def component_is_duplicated(component: dict[str, Any]) -> bool:
     return component_is_duplicated
 
 
+def _add_tools_metadata(syft_sbom: dict[Any, Any]) -> None:
+    """Add Cachi2 as a tool in the metadata section of the SBOM.
+
+    With CycloneDX 1.5, a new format for specifying tools was introduced, and the format from 1.4
+    was marked as deprecated.
+
+    This function aims to support both formats. Notice that, for simplicity reasons, we're ignoring
+    the actual 'tools' key present in the Cachi2 generated SBOM, and the data following the
```

Comment:
Clever. Will this end up being tech debt?

Reply:
Yeah... the kind that might never be paid out. We won't likely change anything about the 'tools' content until (if) we get to the point we're actually adding Cachi2's version number to it. Then this needs to be changed.

On the other hand, the reason I took the simple approach was because I was hoping we could have this fixed ASAP to unblock users from using Yarn in Konflux, but we found a simpler [way](https://github.com/redhat-appstudio/build-definitions/pull/738) to fix it. So I can actually go back and implement this the right way.